### DISCUSSION TITLE
Nft projects

### DISCUSSION THREAD

**Discussion by cosmasnwigbo** [Cyfrin/foundry-full-course-cu#2996] [General]
how to  i engage in the projects

**Comment by EngrPips** [comment]
Hello @cosmasnwigbo, Can you please explain more what you mean by `how to i engage in the projects`

**Reply by cosmasnwigbo** [reply]
i have no idea of what to do with the challenges

**Reply by EngrPips** [reply]
Hello @cosmasnwigbo, Read this discussion https://github.com/Cyfrin/foundry-full-course-cu/discussions/2589, and it should help you understand how to solve the NFT challenge.
